cmd/thanos/sidecar.go

```diff
@@ -305,29 +305,26 @@ func runSidecar(
 }
 
 func validatePrometheus(ctx context.Context, logger log.Logger, m *promMetadata) error {
-	flags := promclient.Flags{
-		TSDBMinTime: model.Duration(2 * time.Hour),
-		TSDBMaxTime: model.Duration(2 * time.Hour),
-	}
+	var (
```

Comment:
I don't think this code change should be part of this PR, or am I missing smth?

Reply:
It is intentionally there, but can split if needed